source/ed/connectors/fusio_parser.cpp

```diff
@@ -342,6 +342,10 @@ std::vector<ed::types::VehicleJourney*> TripsFusioHandler::get_split_vj(Data& da
     std::vector<ed::types::VehicleJourney*> res;
 
     types::MetaVehicleJourney& meta_vj = data.meta_vj_map[row[trip_c]]; //we get a ref on a newly created meta vj
+    // Store the meta vj with its external code, only if this code exists
+    if(is_valid(ext_code_c, row)){
+        gtfs_data.metavj_by_external_code.insert({row[ext_code_c], &meta_vj});
```

Comment:
why do you need this external code map ?

why don't you link the vj by it's id in the files ?